docs/source/workflow.rst

```diff
@@ -180,24 +124,19 @@ Monitoring
 PaaSTA gives you a few `Sensu <https://sensuapp.org/docs/latest/>`_-powered
 monitoring checks for free:
 
-* `setup_marathon_job <generated/paasta_tools.setup_marathon_job.html#module-paasta_tools.setup_marathon_job>`_:
-  Alerts when a Marathon service cannot be deployed or bounced for some reason.
-  It will resolve when a service has been successfully deployed/bounced.
-
-* `check_marathon_services_replication <generated/paasta_tools.check_marathon_services_replication.html>`_:
+* **check_kubernetes_services_replication**:
   runs periodically and sends an alert if fewer than 50% of the requested
-  instances are deployed on a cluster. If the service is registered in Smartstack
-  it will look in Smartstack to count the available instances. Otherwise it
-  counts the number of healthy tasks in Mesos.
+  instances are deployed on a cluster. it will look in Smartstack to count the available instances
+  against the expected amount of instances that should've been deployed via Kubernetes.
```

Comment:
I think we want to go back to the previous text and `s/healthy tasks in Mesos/healthy Pods in Kubernetes` since that's still what we do iirc